docs/imputation.md

```diff
@@ -1,6 +1,138 @@
 # PHGv2 - Imputation
 
-In this document, we will discuss imputation strategies using the 
+In this document, we will discuss imputation strategies using the
 PHG.
 
-_WIP_
\ No newline at end of file
+_WIP_
+
+## Find Paths
+
+FindPaths is the command used to impute paths through a HaplotypeGraph based on a set of read mappings.
+The method uses read mapping files generated by the map-kmers command. Running FindPaths uses the Viterbi algorithm to solve a
+hidden Markov model (HMM) to identify the set of haplotypes most likely to have generated the observed read mappings. A haploid
+path is a single path through the graph. A diploid path is two paths through the graph that jointly explain the read mappings.
+
+### Likely Ancestors
+The term "ancestors" refers to the taxa/assemblies used to construct the haplotype graph used for imputation. The term "ancestors" is used because the model
+considers the haplotypes in the graph to represent the potential ancestral haplotypes from which the samples
+to be imputed were derived. In some scenarios, limiting the number of ancestors used to impute a sample will be useful.
+For example, if some samples were derived from an F1 cross and the individuals used to build the graph include
+those parents, using only those parents for imputation can improve accuracy. Also, because imputing diploid paths
+is more computationally intensive than haploid paths, limiting the number of ancestors used per sample may be necessary to
+control the amount of time required per sample.
+
+To restrict the number of ancestors used, set use-likely-ancestors to true, and provide a value for either max-ancestors
+or min-coverage. For the case where the samples have been derived from a limited number of ancestors setting
+min-coverage to 0.95 or max-ancestors to the expected number of ancestors is a useful strategy. In either case, providing
+a name for the output file saves a record of the ancestors used for each sample and should be checked to make sure the samples
+behaved as expected.
+
+When using this method, ancestors are chosen by first counting the number of reads for a sample that map to each ancestor in the haplotype graph.
+The ancestor with the most mapped reads is chosen. Next the reads not mapping to first chosen ancestor are used to count reads mapping
+to the remaining ancestors and the ancestor with the most reads is selected. This is repeated until either the proportion
+of reads mapping to the selected ancestors >= min-coverage or the number ancestors selected equals max-ancestors. If a
+name is provided for the likely-ancestor-file, the chosen ancestors are written to that file in the order they were chosen
+along with the cumulative coverage, which is the proportion of reads mapping to each ancestor and the previously chosen ones.
+
+### Required Parameters
+
+path-keyfile: the path to a keyfile that contains a list of the read mapping files. The keyfile must have two columns
```

Comment:
surrounding the parameter name with ** to bold it would make reading a bit easier.